.travis.yml

```diff
@@ -22,10 +22,13 @@ before_install:
     - if [ "$TRAVIS_PHP_VERSION" != "hhvm" ]; then phpenv config-rm xdebug.ini; fi;
     - phpunit --self-update
     - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
+    - npm install -g swagger-cli
```

Comment:
IMO it's better as is. Having a `packages.json` in the repository (that doesn't use any NPM dependency) has no sense and it's good to test the Swagger support with the latest available version of swagger-cli (we also install the last version of composer for instance).